doc/README.open_ce_build.md

```diff
@@ -277,6 +280,15 @@ optional arguments:
 ==============================================================================
 ```
 
+For example,
+
+```shell
+    git clone http://github.com/open-ce/open-ce
```

Comment:
We should use the pip install commands here.

Reply:
oh right, will fix.